src/paperwork/paperwork.py

```diff
@@ -73,19 +64,33 @@ def set_locale():
     """
     Enable locale support
     """
-    locale.setlocale(locale.LC_ALL, '')
+    try:
+        locale.setlocale(locale.LC_ALL, '')
+    except locale.Error:
+        # happens e.g. when LC_ALL is set to a nonexisting locale
+        logger.warning("Failed to set LC_ALL, disabling localization")
+        return
 
     got_locales = False
     locales_path = None
-    for (fr_locale_path, locales_path) in LOCALE_PATHS:
-        logger.info("Looking for locales in '%s' ..." % (fr_locale_path))
-        if os.access(fr_locale_path, os.R_OK):
-            logger.info("Will use locales from '%s'" % (locales_path))
-            got_locales = True
-            break
+
+    for locale_base in LOCALE_PATHS:
+        locales_path = os.path.join(locale_base, "locale")
+        logger.debug("Looking for locales in '%s' ..." % locales_path)
```

Comment:
For future code, please use the new formatting operation: logger.debug("Looking for locales in '{}' ...".format(locales_path))
(see https://docs.python.org/3/library/stdtypes.html#old-string-formatting and https://docs.python.org/3/library/stdtypes.html#old-string-formatting )
(this is not a blocking issue here)